tests/Console/Command/DescribeCommandTest.php

```diff
@@ -32,18 +34,31 @@
  */
 final class DescribeCommandTest extends TestCase
 {
+    /**
+     * @var Application
+     */
+    private $application;
+
+    protected function setUp()
+    {
+        $this->application = new Application();
```

Comment:
Setting the property to `null` would remove the only reference to the `Application` instance so PHP should be able to free memory, even if the test class instance itself still exists.